dnf/logging.py

```diff
@@ -152,7 +152,10 @@ def _setup(self, verbose_level, error_level, logdir):
         logger_warnings.addHandler(handler)
 
         lr_logfile = os.path.join(logdir, dnf.const.LOG_LIBREPO)
-        libdnf.repo.LibrepoLog.addHandler(lr_logfile)
+        libdnf.repo.LibrepoLog.addHandler(
+            lr_logfile,
+            True if verbose_level <= 10 else False  # 10 = DEBUG
```

Comment:
Magic value 10. Hmm, a comment says it is DEBUG.
What about:
`True if verbose_level <= DEBUG else False`
or simply
`verbose_level <= DEBUG`

Reply:
Ah, sounds like something that would make sense :smile: